services/templates/service.go

```diff
@@ -20,13 +20,16 @@ var (
 	ErrParseHTMLPart    = errors.New("failed to parse HTMLPart")
 	ErrParseTextPart    = errors.New("failed to parse TextPart")
 	ErrParseSubjectPart = errors.New("failed to parse SubjectPart")
+	ErrDeleteFailed     = errors.New("failed to delete")
```

Comment:
```suggestion
	ErrDeleteFailed     = errors.New("failed to delete template")
```

Reply:
Since we dont need to check the delete marker we dont need this error. I will delete it.